proxy/ParentConsistentHash.cc

```diff
@@ -389,6 +391,7 @@ ParentConsistentHash::markParentUp(ParentResult *result)
 
   ink_atomic_swap(&pRec->failedAt, static_cast<time_t>(0));
   int old_count = ink_atomic_swap(&pRec->failCount, 0);
+  ink_atomic_swap(&pRec->retriers, 0);
```

Comment:
I don't see a point to using `atomic_swap` if the return value isn't checked.

Reply:
@SolidWallOfCode were marking the parent up and just need to reset retriers to 0, the old value from ink_atomic_swap is not really needed.

Reply:
@SolidWallOfCode I removed the ink_atomic's and changed to std::atomic's

Reply:
For 8 byte (or less) values there is no need for an atomic store because it is intrinsically atomic. However, there is `std::atomic<T>::store` for the general case.